liquibase/changeLogs/capture/functions/pruneTimeSeriesData/pruneTimeSeriesData.sql

```diff
@@ -0,0 +1,41 @@
+create or replace function ${AQTS_SCHEMA_NAME}.prune_time_series_data(date timestamp)
+returns void
+language plpgsql
+as $$
+declare
+	month_number integer := extract( month from date );
+	month_name varchar := trim(to_char(date, 'month'));
+begin
+
+	/* drop the partitions */
+	execute format('drop table if exists %I.json_data_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_approvals_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_gap_tolerances_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_grades_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_header_info_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_interpolation_types_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_methods_%s', '${AQTS_SCHEMA_NAME}', month_name);
+	execute format('drop table if exists %I.time_series_qualifiers_%s', '${AQTS_SCHEMA_NAME}', month_name);
+
+	/* recreate the partitions */
+	execute format('create table if not exists %I.json_data_%s partition of %I.json_data for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_approvals_%s partition of %I.time_series_approvals for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_gap_tolerances_%s partition of %I.time_series_gap_tolerances for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_grades_%s partition of %I.time_series_grades for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_header_info_%s partition of %I.time_series_header_info for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_interpolation_types_%s partition of %I.time_series_interpolation_types for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_methods_%s partition of %I.time_series_methods for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+	execute format('create table if not exists %I.time_series_qualifiers_%s partition of %I.time_series_qualifiers for values in (%L)', '${AQTS_SCHEMA_NAME}', month_name, '${AQTS_SCHEMA_NAME}', month_number);
+
+	/* recreate the indexes */
```

Comment:
We need to recreate all of the indexes/constraints on all of the tables.

Reply:
Yes. When we're ready to merge let's discuss strategy.

Reply:
I see what you're saying now.  We had a discussion and resolved the issue.  Also added partitions/constraints/indexes for the time_series_points table.